### PR TITLE
DSND-2871: Fix simple Sonarlint issues

### PR DESCRIPTION
**Pull request by crowlandsCH**
[DSND-2871](https://companieshouse.atlassian.net/browse/DSND-2871)

[DSND-2871]: https://companieshouse.atlassian.net/browse/DSND-2871?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/itest/java/uk/gov/companieshouse/charges/data/steps/ChargesApiSteps.java [hunk 1]
```diff
@@ -363,25 +362,25 @@ public void i_send_delete_request_with_company_number(String companyNumber, Stri
...
     @Then("the CHS Kafka API is invoked successfully with delete event payload {string}")
     public void chs_kafka_api_invoked(String payload) throws IOException {
 
         File file = new FileSystemResource("src/itest/resources/payload/input/" + payload + ".json").getFile();
         List<ServeEvent> serverEvents = WiremockTestConfig.getServeEvents();
-        assertThat(serverEvents.size()).isEqualTo(1);
-        assertThat(serverEvents.isEmpty()).isFalse();
-        String requestReceived = serverEvents.get(0).getRequest().getBodyAsString();
+        assertThat(serverEvents).hasSize(1);
+        assertThat(serverEvents).isNotEmpty();
```
Comment (johncook-ch) [inline src/itest/java/uk/gov/companieshouse/charges/data/steps/ChargesApiSteps.java:382]: The second line is semantically the same as the first

Comment (crowlandsCH) [reply src/itest/java/uk/gov/companieshouse/charges/data/steps/ChargesApiSteps.java]: good point, I've removed it.

**Review by johncook-ch — APPROVED** [review]

**Review by cgregoryCH — APPROVED** [review]

**Review by sthompsonCH — APPROVED** [review]

File: src/itest/java/uk/gov/companieshouse/charges/data/steps/ChargesApiSteps.java [hunk 2]
```diff
@@ -64,17 +64,16 @@ public class ChargesApiSteps {
 
     String companyNumber = "08124207";
     String chargeId = "AbRiNTU3NjNjZWI1Y2YxMzkzYWY3MzQ0YzVlOTg4ZGVhZTBkYWI4Ng==";
-    String insolvency_cases_happy_path_input = "Insolvency_cases_Happy_Path_input";
-    String invalid_payload = "Invalid_payload";
-    String x_request_value = "5234234234";
-    String x_request_id = "x-request-id";
+    String insolvencyCasesHappyPathInput = "Insolvency_cases_Happy_Path_input";
```
Comment (johncook-ch) [inline src/itest/java/uk/gov/companieshouse/charges/data/steps/ChargesApiSteps.java:67]: These should all really be `private static final` strings with variable names as upper case/snake case

Comment (crowlandsCH) [reply src/itest/java/uk/gov/companieshouse/charges/data/steps/ChargesApiSteps.java]: Addressed in most recent commit

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by cgregoryCH — APPROVED** [review]

**Review by johncook-ch — APPROVED** [review]

**Pull request merged**